Source/WebCore/page/LocalFrame.cpp

```diff
@@ -376,7 +376,7 @@ void LocalFrame::orientationChanged()
 IntDegrees LocalFrame::orientation() const
 {
     if (auto* page = this->page())
-        return page->chrome().client().deviceOrientation();
+        return page->chrome().client().deviceOrientation() * -1;
```

Comment:
I doubt we should be changing this. This is code pre-dating the screen orientation API.

Reply:
Right, but we report the opposite angles :( Not sure what to do here.

Reply:
You say "angle" and I am confused because there is:
(1) window.orientation, which is legacy and for which we shouldn't change behavior.
(2) window.screen.orientation.angle, which is the new API and which value is deduced from window.screen.orientation.type

I don't think we should change the values returned by (1), even if they are reversed compared to Android. Since this is long-standing behavior, I believe backward compatibility is more important. As far as I know, the code you are modifying in this function should only impact (1), which is why I am surprised.